modules/phase_field/src/utils/Euler2RGB.C

```diff
@@ -289,30 +289,37 @@ euler2RGB(unsigned int sd, Real phi1, Real PHI, Real phi2, unsigned int phase, u
       // Increment to next symmetry operator if not in SST
       else
         index++;
+
+      // Check for solution
+      mooseAssert(!(index == nsym),
```

Comment:
`!(index == nsym)` ? That's a peculiar way to write `index != nsym`, but it's fine

Reply:
Yep, I was a little tired when I did that.  I will fix it and push it to the branch.